src/build/optimize-pipe.ts

```diff
@@ -10,9 +10,51 @@
 
 import * as stream from 'stream';
 
+import * as gulpif from 'gulp-if';
```

Comment:
Oh, 'stream' is an internal-to-node module. I thought we wanted to separate those.